frontend/src/pages/modelRegistrySettings/CreateMRSecureDBSection.tsx

```diff
@@ -25,22 +27,25 @@ interface CreateMRSecureDBSectionProps {
   secureDBInfo: SecureDBInfo;
   modelRegistryNamespace: string;
   nameDesc: { name: string };
-  existingCertKeys: string[];
-  existingCertConfigMaps: string[];
-  existingCertSecrets: string[];
+  existingCertConfigMaps: ConfigSecretItem[];
+  existingCertSecrets: ConfigSecretItem[];
   setSecureDBInfo: (info: SecureDBInfo) => void;
 }
 
 export const CreateMRSecureDBSection: React.FC<CreateMRSecureDBSectionProps> = ({
   secureDBInfo,
   modelRegistryNamespace,
   nameDesc,
-  existingCertKeys,
   existingCertConfigMaps,
   existingCertSecrets,
   setSecureDBInfo,
 }) => {
-  const [searchValue, setSearchValue] = useState('');
+  const [configSecretName, setConfigSecretName] = useState('');
```

Comment:
I was a little confused by the name of this since it's the search input and not the selected value, can we rename it to something with `search` in the name?